app/views/shared/_navbar.html.erb

```diff
@@ -8,8 +8,10 @@
           <li><%= link_to "Demo Nights", admin_demo_nights_path %></li>
           <% if !current_demo_night? %>
             <li><%= link_to "New Demo Night", new_admin_demo_night_path %></li>
+          <% else %>
+            <li><%= link_to "Current Projects", admin_demo_night_path(DemoNight.current) %></li>
           <% end %>
-          <li><%= link_to "Current Projects", admin_projects_path %></li>
```

Comment:
@notmarkmiranda Not that I can think of.

Did a universal search and think there are two other changes to make:

- Remove the link to "All Projects" from the Admin Projects Show.
- Delete the test `/spec/features/admin_can_view_projects_index_spec.rb`. Believe we're already checking to see that an Admin can view a list of projects in `/spec/features/admin_can_view_demo_night_spec.rb`.